### DISCUSSION TITLE
Has anyone tried these for Windows for ARM?

### DISCUSSION THREAD

**Discussion by abufrejoval** [virtio-win/kvm-guest-drivers-windows#1088] [General]
In a perfect world this should just work... But before I dive into even trying to set up a build environment, I thought I'd ask...

Running KVM/Proxmox on AARCH64 and virt-drivers might actually provide a better experience than 'native/un-accelerated' drivers for Windows there, apart from the VM benefits.

**Comment by vrozenfe** [comment]
I have some experience running HLK  on ARM64 vioscsi and vigpudo device drivers. There were some tests that failed but over all it doesn't look bad at all. 
Vadim.

**Comment by YanVugenfirer** [comment]
Yes, people are using those drivers. Origina issue with a lot of discussions: https://github.com/virtio-win/kvm-guest-drivers-windows/issues/177

And also Alexander Graph published a post about running Windows with virtio-win drivers on top of Apple M1.

**Comment by zyimin** [comment]
I started Windows 11 on ARM with stratovirt. https://gitee.com/openeuler/stratovirt
